sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -240,6 +273,16 @@ def __init__(self, allowed_origins, allowed_methods, **kwargs):
         self.exposed_headers = ','.join(kwargs.get('exposed_headers', []))
         self.max_age_in_seconds = kwargs.get('max_age_in_seconds', 0)
 
+    @classmethod
+    def _from_generated(cls, generated):
+        return cls(
+            [generated.allowed_origins],
```

Comment:
Is this not creating a list of a list?

Reply:
That's actually the opposite, that's creating the right think :).
We override the behavior of the original "allowed_origins" (a str) with a list, and we join it. To avoid this join, I wrapped in a list of one element, so we get what we expect.
See __init__

Reply:
Ah, sweet. I didn't realize this property was a str in the generated model :)